hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -534,7 +534,9 @@ public HRegionServer(final Configuration conf) throws IOException {
       regionServerAccounting = new RegionServerAccounting(conf);
 
       blockCache = BlockCacheFactory.createBlockCache(conf);
-      DataTieringManager.instantiate(onlineRegions);
+      if (DataTieringManager.isDataTieringFeatureEnabled(conf)) {
+        DataTieringManager.instantiate(conf, onlineRegions);
+      }
```

Comment:
Since we are not instantiating the DataTieringManager itself when global configuration is not enabled, do we need to restart every time we change that global parameter?